io/parser.js

```diff
@@ -987,6 +1098,47 @@ X.parser.reslice2 = function(_sliceOrigin, _sliceXYSpacing, _sliceNormal, _color
   return sliceXY;
 };
 
+/**
+ * Compute plane origin and two points based on view type.
+ * @param {number} xyz view type
+ * @param {Object} bounds of the plane
+ * @param {goog.vec.Vec3} center center of the plane
+ * @param {goog.vec.Vec3} volumeCenter center of the volume
+ */
+ X.parser.computePlaneOrigins = function(xyz, bounds, center, volumeCenter){
+  var planeSource = {};
+  var origin = [];
+  var point1 = [];
+  var point2 = [];
+  var offset = [0,0,0];
+
+  // Offset based on the center of the image and how far from the
+  // reslice cursor is. This allows us to capture the whole image even
+  for (let i = 0; i < 3; i++) {
```

Comment:
Stray `let` among ES5-compliant code.
```suggestion
  for (var i = 0; i < 3; i++) {
```